petab/v2/core.py

```diff
@@ -219,12 +242,24 @@ class Change(BaseModel):
 
     A change to the model or model state, corresponding to an individual
     row of the PEtab conditions table.
+
+    >>> Change(
+    ...     target_id="k1",
+    ...     operation_type=OperationType.SET_CURRENT_VALUE,
+    ...     target_value="10",
```

Comment:
Using `"10"` is fine but it looks like int/float is handled? Is this to stress that the type is an expression? Then maybe using e.g. `k1+3` makes the point better.

Reply:
That's to test that strings are accepted/converted.